functions/exercises.py

```diff
@@ -313,7 +367,8 @@ def reorder_hyphen_sequence(string):
     :return:
     """
     # INSERT YOUR CODE HERE
-    #
+    arr = string.split("-")
+    return "-".join(sorted(arr))

```

Comment:
```suggestion
    return "-".join(sorted(string.split("-")))
```

You are definning a variable that is only used once...
The code looks simple enough to be a one liner